core/templates/base.html

```diff
@@ -5,6 +5,8 @@
     {% block extra_head %}
     {% endblock %}
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
let's add this line to the base. it will adjust for small screens automatically i think:
`    <meta name="viewport" content="width=device-width, initial-scale=1.0">`

Reply:
`{% load static %}
    <link rel="stylesheet" href="{% static 'css/main.css' %}">`

Reply:
done!